jupyterhub/app.py

```diff
@@ -1311,6 +1311,13 @@ def cleanup(self):
         """Shutdown our various subprocesses and cleanup runtime files."""
```

Comment:
Perhaps update here: """Shutdown our managed services, various subprocesses, and cleanup runtime files."""